client/src/components/EditDetails.jsx

```diff
@@ -1,60 +1,68 @@
-import React from 'react';
-import { Dropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'reactstrap';
-import '!style-loader!css-loader!bootstrap/dist/css/bootstrap.css';
-import axios from 'axios';
+import React from 'react'
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from 'reactstrap'
+import '!style-loader!css-loader!bootstrap/dist/css/bootstrap.css'
+import axios from 'axios'
+
+const Months = Array.from(Array(12)).map(i => i + 1)
```

Comment:
Lift Months and Days calculation to be constants at the top of the file, so we don't have to calculate it every time validation function is triggered. Also we could shorten the array creation using `Array.from()`

I think for birth month and dat validation, we could probably also do `0 < birthMonth < 13` and `0 < birthDay < 32`